assets/base/i18n/en.json

```diff
@@ -2757,6 +2768,7 @@
   "status_dropdown.set_away": "Away",
   "status_dropdown.set_dnd": "Do Not Disturb",
   "status_dropdown.set_dnd.extra": "Disables Desktop and Push Notifications",
+  "status_dropdown.set_ooo": "Out Of Office",
```

Comment:
This string already exists below. Also, we shouldn't change non-mobile translation strings in this repo since those will get overwritten by the translation server

Reply:
Yeh, my bad.  Must have been from moving back and forth between mobile and webapp implementation